qa/suites/upgrade/quincy-x/stress-split/1-start.yaml

```diff
@@ -0,0 +1,122 @@
+tasks:
+- install:
+    branch: pacific
+    exclude_packages:
+      - ceph-volume
+
+- cephadm:
+    image: docker.io/ceph/daemon-base:latest-pacific
+    cephadm_branch: pacific
+    cephadm_git_url: https://github.com/ceph/ceph
+    conf:
+      osd:
+        #set config option for which cls modules are allowed to be loaded / used
+        osd_class_load_list: "*"
+        osd_class_default_list: "*"
+
+- cephadm.shell:
+    mon.a:
+      - ceph fs volume create foo
+      - ceph config set mon mon_warn_on_insecure_global_id_reclaim false --force
```

Comment:
can remove these mon_warn lines, shouldn't be needed in quincy

Reply:
removed two `mon_warn lines`